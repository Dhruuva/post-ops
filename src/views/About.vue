<template lang="pug">
  v-main.grey.lighten-2
    v-layout.ma-3( row wrap )
      v-flex.d-md-inline( xs8 )
        v-sheet.pa-5(elevation="5" )
          v-avatar(class="profile"  color="grey" size="164" tile)
            v-img( src="@/assets/avatars/man.png" alt='Avatar')
          v-text-field(label="Name " single-line)
          v-text-field(label="Email " single-line)
          v-btn.ma-2(small ) Sample
         
      v-flex.d-md-inline( xs8 )
        v-sheet(height='50' color="green lighten-3" elevation="5" )   
</template>
