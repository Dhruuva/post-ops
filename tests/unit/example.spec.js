import Vue from 'vue'
import { createLocalVue, mount } from '@vue/test-utils'
import About from '@/views/About'
import Vuetify from 'vuetify'
Vue.config.silent = true;
describe('About.vue', () => {
	const localVue = createLocalVue()
    let vuetify
	let $vuetify
     beforeEach(() => {
	    vuetify = new Vuetify()
	    $vuetify = new Vuetify()
	 })

	 it('About.vue is work', () => {
	    
	    const wrapper = mount(About, {
	    	localVue,
     		vuetify,
     		$vuetify

	    })

	    //expect(wrapper.text()).toMatch("fhf")
	    expect(wrapper.html()).toMatchSnapshot()
	})
})
