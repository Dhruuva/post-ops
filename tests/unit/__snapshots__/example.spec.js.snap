// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue About.vue is work 1`] = `
<v-main class="grey lighten-2">
  <v-layout row="row" wrap="wrap" class="ma-3">
    <v-flex xs8="xs8" class="d-md-inline">
      <v-sheet elevation="5" class="pa-5">
        <v-avatar color="grey" size="164" tile="tile" class="profile">
          <v-img src="@/assets/avatars/man.png" alt="Avatar"></v-img>
        </v-avatar>
        <v-text-field label="Name " single-line="single-line"></v-text-field>
        <v-text-field label="Email " single-line="single-line"></v-text-field>
        <v-btn small="small" class="ma-2">Sample</v-btn>
      </v-sheet>
    </v-flex>
    <v-flex xs8="xs8" class="d-md-inline">
      <v-sheet height="50" color="green lighten-3" elevation="5"></v-sheet>
    </v-flex>
  </v-layout>
</v-main>
`;
